elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/MultiplexTransaction.java

```diff
@@ -140,6 +141,18 @@ protected DataStoreTransaction getTransaction(Class<?> cls) {
         return transaction;
     }
 
+    protected DataStoreTransaction getRelationTransaction(Object object, String relationName, RequestScope scope) {
```

Comment:
Why not just pass in the dictionary or something here.  THen you don't need to cast, etc.